client/components/pagination/docs/example.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Pagination from 'components/pagination';
+
+class PaginationExample extends Component {
+	state = {
+		page: 1,
+	}
+
+	updatePage = page => {
+		this.setState( { page } );
+	}
```

Comment:
Not a request, but just to point out that this could also be written with an implicit return:
```es6
updatePage = page => this.setState( { page } );
```